### PR TITLE
Admin Style Upgrade + Fully working button edit

### PR DESCRIPTION
**Pull request by SamHarbi**

### CODE REVIEW

**Review by ScottyDog5 — COMMENTED** [review]
Looks good, see no reason not to merge it

File: Software_Code/Admin/Admin_page.php [hunk 1]
```diff
@@ -80,6 +80,70 @@ function createNewUser($name, $pass)
     createNewUser($newUsername, $newPassword);
 }
```
Comment (DanielJakubek) [inline Software_Code/Admin/Admin_page.php:82]: The logic needs comments to explain what is happening,

File: Software_Code/Admin/template.php [hunk 2]
```diff
@@ -0,0 +1,273 @@
+<!doctype html>
+<html lang="en">
```
Comment (DanielJakubek) [inline Software_Code/Admin/template.php]: What is this and what purpose does it serve? It seems to be using old map code too?

**Review by DanielJakubek — CHANGES_REQUESTED** [review]
Looks good apart from the lack of comments in Admin_page.php and template.php seems to serve no purpose?

**Review by DanielJakubek — APPROVED** [review]

**Comment by DanielJakubek** [comment]
Cool, fixed the issues. Add the comments in the branch.

**Pull request merged**